lib/footerComponent.js

```diff
@@ -0,0 +1,9 @@
+import { createElement } from "./elements";
```

Comment:
Sollte das nicht so sein?

```suggestion
import { createElement } from "./elements.js";
```